mounted_file_system.go

```diff
@@ -47,3 +50,17 @@ func (mfs *MountedFileSystem) Join(ctx context.Context) error {
 		return ctx.Err()
 	}
 }
+
+// GetFuseContext implements the equiv. of FUSE-C get_fuse_context() and thus
```

Comment:
The C equivalent is fuse_get_context, not get_fuse_context

Reply:
Indeed. I forced-push on the branch of my PR to correct this.